src/components/MnemonicWord/Circle.ts

```diff
@@ -1,15 +1,9 @@
 import styled from 'styled-components'
-import Box from '../Box'
 
-export default styled(Box).attrs(props => ({
-  height: 4,
-  width: 3,
-  fontSize: 1,
-  borderRadius: 6,
-  mx: 2,
-  ...props
-}))`
-  display: flex;
-  justify-content: center;
-  align-items: center;
+export default styled.div`
+  width: 1.5em;
+  height: 1.5em;
+  font-size: 1.125em;
+  text-align: center;
+  line-height: 1.5em;
```

Comment:
I got some weird DOM behaviour around the Mnemonic word part, so thought it could use a quick clean up.